exec/java-exec/src/test/java/org/apache/drill/exec/sql/TestInfoSchema.java

```diff
@@ -351,4 +359,51 @@ public void showFilesWithDefaultSchema() throws Exception{
     test("USE dfs_test.`default`");
     test("SHOW FILES FROM `/tmp`");
   }
+
+  @Test
+  public void describeSchemaSyntax() throws Exception {
+    test("describe schema dfs_test");
+    test("describe schema dfs_test.`default`");
+    test("describe database dfs_test.`default`");
+  }
+
+  @Test
+  public void describeSchemaOutputForDefaultWorkspace() throws Exception {
+    String properties = Resources.toString(Resources.getResource("describe_schema_output.json"), Charsets.UTF_8).replace("\r", "");
+    testBuilder()
+        .sqlQuery("describe schema dfs_test")
+        .unOrdered()
+        .baselineColumns("schema", "properties")
+        .baselineValues("dfs_test", properties)
```

Comment:
I'm not sure if it's a good idea to put the complete "properties" as the expected output in unit test. Doing that makes this unit test prone to failure, in case we change StoragePlugin interface, and add / delete contents in the schema properties. This is similar to Explain Plan output. If we capture everything, it becomes annoying in the future, since any slight change to the planer might fail the unit test.

In stead of capturing everything, can you just verify some "stable" part of properties, like "location" etc, to make this unit test less likely to fail in the future?